### PR TITLE
add amgov answer key strategy

### PR DESCRIPTION
**Pull request by brittany-johnson**
related card https://github.com/openstax/cnx-recipes/issues/3506

### CODE REVIEW

File: lib/locales/en.yml [hunk 1]
```diff
@@ -23,6 +23,10 @@ en:
   eoc_key_concepts: Key Concepts
   eoc_key_equations: Key Equations
   eoc_suggested_reading: Suggestions for Further Study
+  eoc: {
+    review-questions: Review Questions,
+    critical-thinking: Critical Thinking Questions
+  }
```
Comment (jpslav) [inline lib/locales/en.yml:29]: Why are these added if they aren't used in this PR?

Comment (brittany-johnson) [reply lib/locales/en.yml:29]: @jpslav In my recipe, I'm using `MoveExercisesToEOC.v1` and passing these values to it. I included it in this PR because both of these sections have solutions in my answer key.

File: lib/kitchen/directions/move_solutions_to_answer_key/strategies/american_government.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        append_solution_area(section_solutions_set, append_to)
+      end
+
+      def append_solution_area(solutions, append_to)
+        solutions.each do |solution|
+          append_to.add_child(solution.raw)
+        end
+      end
+    end
+  end
```
Comment (jpslav) [inline lib/kitchen/directions/move_solutions_to_answer_key/strategies/american_government.rb:29]: Actually can't these two methods be rewritten as one method:

```ruby
def bake_section(chapter:, append_to:, klass:)
  chapter.search(".#{klass} [data-type='solution']").each do |solution|
    append_to.append(child: solution.cut.to_s)
  end
end
```

**Review by jpslav — CHANGES_REQUESTED** [review]

**Review by jpslav — APPROVED** [review]
